src/main.cpp

```diff
@@ -121,11 +121,12 @@ static bool doIdSelect()
 	int flen;
 	char *cBuf;
 
-	FILE *f = fopen("sd:/hbl2hbc.txt","rb");
+	//use open homebrew launcher (OHBC) instead of homebrew launcher (LULZ)
+	//set title value here so it doesn't need to get changed twice (or more often) for future modifications
+	titleToBoot=0x4F484243; 
+	FILE *f = fopen("sd:/wiiu/apps/hbl2hbc/hbl2hbc.txt","rb"); //load the txt from the app folder instead of the sd card root
```

Comment:
(so it doesn't clutter the sd root or confuse people who think of adding the hbl2hbl.txt to the app's folder)